cockroachdb/example/cert-manager/disable-backup.yaml

```diff
@@ -0,0 +1,17 @@
+# Remove Service Account from StatefulSet
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: cockroachdb
+spec:
+  template:
+    spec:
+      $patch: delete
+      serviceAccountName: cockroachdb
```

Comment:
Do we really need to delete the service account? Does it fail if we leave it in?